interface/src/avatar/MyAvatar.cpp

```diff
@@ -5159,16 +5234,46 @@ bool MyAvatar::getIsInWalkingState() const {
     return _isInWalkingState;
 }
 
+// Determine if the user is sitting in the real world.
 bool MyAvatar::getIsInSittingState() const {
     return _isInSittingState.get();
 }
 
+// Deprecated, will be removed.
 MyAvatar::SitStandModelType MyAvatar::getUserRecenterModel() const {
-    return _userRecenterModel.get();
+    qCDebug(interfaceapp)
+        << "MyAvatar.getUserRecenterModel is deprecated and will be removed.";
+
+    // The legacy SitStandModelType corresponding to each AllowAvatarLeaningPreference.
+    std::array<SitStandModelType, static_cast<uint>(AllowAvatarLeaningPreference::Count)> legacySitStandModels = {
+        SitStandModelType::Auto,           // AllowAvatarLeaningPreference::WhenUserIsStanding
+        SitStandModelType::ForceStand,     // AllowAvatarLeaningPreference::Always
+        SitStandModelType::ForceSit,       // AllowAvatarLeaningPreference::Never
+        SitStandModelType::DisableHMDLean  // AllowAvatarLeaningPreference::AlwaysNoRecenter
+    };
+
+    return legacySitStandModels[static_cast<uint>(_allowAvatarLeaningPreference.get())];
 }
 
+// Deprecated, will be removed.
 bool MyAvatar::getIsSitStandStateLocked() const {
-    return _lockSitStandState.get();
+    qCDebug(interfaceapp) << "MyAvatar.getIsSitStandStateLocked is deprecated and will be removed.  If you need it, please "
+                             "contact the developers.  See also: MyAvatar.getUserRecenterModel.";
```

Comment:
A remaining occurrence of "contact the developers".

Reply:
Thanks, fixed in a54a40dc2cb1555b4973489d83e57d449bc19c9c